WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -7,6 +7,7 @@ enum FeatureFlag: Int {
     case saveForLater
     case gifSupportInReaderDetail
     case extractNotifications
+    case customDomainForBusinessPlan
```

Comment:
I know the project description relates to Business Plans, but this feature isn't just for Business Plans per se. It's more related to Automated Transfers. Can we change to this something like `automatedTransfersCustomDomain`?

Reply:
Done